notebooks/Annotation and Continuous Learning Overview.ipynb

```diff
@@ -64,11 +64,11 @@
    "source": [
     "# API Endpoint and Credentials\n",
     "host_url = \"https://api.monai.ngc.nvidia.com\"\n",
-    "ngc_api_key = os.environ.get('MONAI_API_KEY')\n",
+    "ngc_api_key = os.environ.get(\"MONAI_API_KEY\", \"<YOUR_API_KEY>\")  # we recommend using environment variables for API keys, but you can also hardcode them here\n",
```

Comment:
Can simply these to <YOUR_NGC_API_KEY>?

Reply:
Hi @zephyrie because we mention using env variable `MONAI_API_KEY` to keep secret is safer, so I have both flavors in this line: the secure way to load from env var, and the insecure way to put the token in the notebook. If you think the insecure way is good enough, I can go that way.

Reply:
Hi @zephyrie , @yiheng-wang-nv needs this merge to unblock him. I will create a separate ticket for this issue

Reply:
Issue in https://github.com/NVIDIA/monai-cloud-api/issues/79